### PR TITLE
feat(hops-jest-preset): add jest preset

### PR DESCRIPTION
**Pull request by robin-drexler**

### CODE REVIEW

File: packages/jest-preset/jest-preset.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "moduleNameMapper": {
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/node_modules/jest-preset-hops/fileMock.js",
```
Comment (dmbch) [inline packages/jest-preset/jest-preset.json]: I think we ought to support the same file types here and in `url`/`file` loaders. That would mean these:

`(png|gif|jpe?g|webp|html|svg|((o|t)tf)|woff2?|ico)`

File: packages/jest-preset/jest-preset.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "moduleNameMapper": {
```
Comment (dmbch) [inline packages/jest-preset/jest-preset.json]: Don't we also have to mock `tpl` loader, i.e. provide a way to require a file with `tpl` extension and have a function returned?

File: packages/jest-preset/jest-preset.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/node_modules/jest-preset-hops/fileMock.js",
+    "^.+\\.css$": "identity-obj-proxy"
+  },
+  "transform": {
+    "^.+\\.(js|jsx)$": "<rootDir>/node_modules/jest-preset-hops/transform.js"
```
Comment (dmbch) [inline packages/jest-preset/jest-preset.json]: Aren't we missing `mjs` here?

File: packages/jest-preset/transform.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+var babelJest = require('babel-jest');
+
+// eslint-disable-next-line import/no-internal-modules
```
Comment (dmbch) [inline packages/jest-preset/transform.js]: I'm not sure we need this in with semistandard :)

**Comment by robin-drexler** [comment]
@dmbch addressed your findings, could you PTAL again? Cheers! :)

**Pull request merged**